docs/Spacer.md

```diff
@@ -0,0 +1,41 @@
+# Spacer
+
+An invisible utility component that acts as a spacer element in various layouts. It works with flexbox sizing or fixed sizing.
+
+<div style="width: 100%; text-align:center">
+    <img width="40%" alt="Spacer used in Drawer Body" src="./images/spacer.png"><br/>
```

Comment:
This may not be the _best_ way to demonstrate the spacer component, only because if you drop a spacer into the drawer body to do that, it actually won't have any effect.

It might be worth adding a callout somewhere in here to mention that using flex elements (like Spacer) inside of a ScrollView will not have an effect unless the parent has a fixed height.

Reply:
that explained why I spent 2 hours trying to add spacer to the showcase drawer and it didn't work 🙃

are there other places in the showcase where I can add spacer, and use that as a good screenshot for this readme?

Reply:
There are ways of making it work in a ScrollView (I'm pretty sure), but it's a little more complicated than just dropping it into the body (I think you have to tweak some styles for the ScrollView scrollContentStyles or something like that, but someone using our Drawer wouldn't have access to that). 

As far as somewhere else you could put it, I'm a little short on ideas. I could think of some ways I would use it to build things, but our showcase right now is just using our components which already handle a lot of the layout bits. 

One place where I have actually used it in practice is separating cards in a multi-column layout. I put a fixed size spacer (with width and height) between the cards. In multi column, the width effectively sets the gutter between them, and when the layout responds down to single column, the height creates a space between them (it's cool because you don't have to do any annoying stuff with media queries and the margins). You could maybe do something like that and show the demo in landscape, but it might be overkill. It might be easier just to make a simple component like a card header with a previous/next arrow in the corners (and toss it on the empty state card, like maybe it has multiple views):

```[ < ___________________________ >]```

Reply:
So a pseudo carousel like this, with a spacer inside a card?
![image](https://user-images.githubusercontent.com/8997218/88922195-c9e09c80-d23d-11ea-873c-1517b4ce9d3f.png)

Hmmmmmmmm... on mobile, it is rare to have onPress interactions to switch between different stages, usually it is a mix of onPress and on swipe.

Reply:
I was thinking in some kind of header, not necessarily the body...I agree it's kind of a stretch. 
![image](https://user-images.githubusercontent.com/29152776/88923631-0614fc80-d240-11ea-88a8-c4d37a411dc8.png)

Maybe we just show a generic example with colored blocks (from the storybook)